src/router.rs

```diff
@@ -146,6 +147,32 @@ impl ResourceInfo {
         }
         false
     }
+
+    /// Check if application contains matching route.
+    ///
+    /// This method does take `prefix` into account
+    /// but behaves like `has_route` in case `prefix` is not set in the router.
+    ///
+    /// For example if prefix is `/test` and router contains route `/name`, the
+    /// following path would be recognizable `/test/name` and `has_prefixed_route()` call
+    /// would return `true`.
+    /// It will not match against prefix in case it's not given. For example for `/name`
```

Comment:
I'm not sure I follow this comment, what exactly you wanted to say here?
Did you mean that path `/name` would match route `/name` when prefix `/test` is omitted?